.vortex/tests/bats/_helper.deployment.bash

```diff
@@ -153,6 +153,13 @@ setup_robo_fixture() {
   fixture_prepare_dir "${HOME}/.composer/vendor/bin"
   touch "${HOME}/.composer/vendor/bin/robo"
   chmod +x "${HOME}/.composer/vendor/bin/robo"
+  
+  # Also create a mock for git-artifact
+  touch "${HOME}/.composer/vendor/bin/git-artifact"
+  chmod +x "${HOME}/.composer/vendor/bin/git-artifact"
+  
+  # Ensure mocks are picked up by tests
+  export PATH="${HOME}/.composer/vendor/bin:$PATH"
```

Comment:
@skipper-vp 
can you please remove this line and try running without it locally and in the CI here. This functionality is already handled in `bats-helpers` - https://github.com/drevops/bats-helpers/blob/main/src/mock.bash#L232